packages/components/src/mobile/bottom-sheet/range-cell.native.js

```diff
@@ -37,8 +41,23 @@ class BottomSheetRangeCell extends Component {
 		}
 	}
 
+	componentDidMount() {
+		AppState.addEventListener( 'change', this.handleChangePixelRatio );
+	}
+
 	componentWillUnmount() {
 		this.handleToggleFocus();
+		AppState.removeEventListener( 'change', this.handleChangePixelRatio );
+	}
+
+	getFontScale() {
+		return PixelRatio.getFontScale() < 1 ? 1 : PixelRatio.getFontScale();
+	}
+
+	handleChangePixelRatio( nextAppState ) {
+		if ( nextAppState === 'active' ) {
+			this.setState( { fontScale: this.getFontScale() } );
+		}
 	}
```

Comment:
Honestly, I think that it's worth to save that info in redux store, because it can be useful in the future to control other components 🤔

Reply:
It does sound like it would be helpful! But maybe it's too much for this PR.